### PR TITLE
add tf_config

### LINKED ISSUE #1472: How to edit keras_policy tf's configproto
**Rasa Core version**:
rasa-core 0.11.4

**Python version**: 
python 3.6

**Operating system** (windows, osx, ...):
unbuntu

**Issue**:
Hello, some questions about keras_policy.py. In the source code, can't add tf config.
```
with self.graph.as_default():
            self.session = tf.Session()
            with self.session.as_default():
                if self.model is None:
                    self.model = self.model_architecture(shuffled_X.shape[1:],
                                                         shuffled_y.shape[1:])

                validation_split = kwargs.get("validation_split", 0.0)
                logger.info("Fitting model with {} total samples and a validation "
                            "split of {}".format(training_data.num_examples(),
                                                 validation_split))
                # filter out kwargs that cannot be passed to fit
                params = self._get_valid_params(self.model.fit, **kwargs)

                self.model.fit(shuffled_X, shuffled_y, **params)
                # the default parameter for epochs in keras fit is 1
                self.current_epoch = kwargs.get("epochs", 1)
                logger.info("Done fitting keras policy model")
```
And everytime I create custome policy should reset train function, like
```
with self.graph.as_default():
            self.session = tf.Session(config = config)
            with self.session.as_default():
                if self.model is None:
                    self.model = self.model_architecture(shuffled_X.shape[1:],
                                                         shuffled_y.shape[1:])

                validation_split = kwargs.get("validation_split", 0.0)
                logger.info("Fitting model with {} total samples and a validation "
                            "split of {}".format(training_data.num_examples(),
                                                 validation_split))
                # filter out kwargs that cannot be passed to fit
                params = self._get_valid_params(self.model.fit, **kwargs)

                self.model.fit(shuffled_X, shuffled_y, **params)
                # the default parameter for epochs in keras fit is 1
                self.current_epoch = kwargs.get("epochs", 1)
                logger.info("Done fitting keras policy model")
```
About config:
```
config = tf.ConfigProto(device_count={"CPU":40},
inter_op_parallelism_threads=0,
intra_op_parallelism_threads=0
)  
config.gpu_options.allow_growth = True 
```
So can provide support interface? Or if it useful, I could sumbit a pr.

### PR DESCRIPTION
**Pull request by GaoQ1**
**Proposed changes**:
- add support tf_config
- fixes #1472

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by Ghostvv** [comment]
@GaoQ1 there is `2` blank lines before line 1433, but should be only `1`

File: rasa_core/policies/embedding_policy.py [hunk 1]
```diff
@@ -1404,6 +1425,11 @@ def persist(self, path: Text) -> None:
         with io.open(dump_path, 'wb') as f:
             pickle.dump(self.encoded_all_actions, f)
 
+        dump_config_path = os.path.join(path, file_name + ".config.pkl")
+        with io.open(dump_config_path, 'wb') as f:
+            pickle.dump(self.config, f)
+
+
```
Comment (Ghostvv) [inline rasa_core/policies/embedding_policy.py]: @GaoQ1 there are `2` blank lines before line 1433, but should be only `1` here

**Comment by Ghostvv** [comment]
probably there are some commits from @gaoquan, who didn't sign CLA

**Comment by Ghostvv** [comment]
@GaoQ1 do you plan to create a new PR with this functionality?

**Comment by GaoQ1** [comment]
Yes, let me resubmit

**Pull request closed without merging**